contrib/nydusd-http-server/README.md

```diff
@@ -0,0 +1,105 @@
+# Http server for nydusd
+A http server based on Rocket to serve static file for nydusd.
+
+
+## APIs supported
+support the following APIs:
+```bash
+HEAD /$namespace/$repo/blobs/sha256:xxx #get blob size
+GET /$namespace/$repo/blobs/sha256:xxx #get blob range data
+```
+### Existing Layers
+```
+HEAD /v2/<name>/blobs/<digest>
+```
+On Success: OK
+```
+200 OK
+Content-Length: <length of blob>
+Docker-Content-Digest: <digest>
+
+```
+### Fetch Blob
+```
+GET /v2/<name>/blobs/<digest>
+Host: <registry host>
+Authorization: <scheme> <token>
+```
+On Success: OK
+```
+200 OK
+Content-Length: <length>
+Docker-Content-Digest: <digest>
+Content-Type: application/octet-stream
+
+<blob binary data>
+```
+On Failure: Not Found
+```
+404 Not Found
+```
+### Fetch Blob Part
+```
+GET /v2/<name>/blobs/<digest>
+Host: <registry host>
+Authorization: <scheme> <token>
+Range: bytes=<start>-<end>
+```
+On Success: OK
+```
+200 OK
+Content-Length: <length>
+Docker-Content-Digest: <digest>
+Content-Range: bytes <start>-<end>/<size>
+Content-Type: application/octet-stream
+
+<blob binary data>
+```
+On Failure: Not Found
+```
+404 Not Found
+```
+
+On Failure: Range Not Satisfiable
+```
+416 Range Not Satisfiable
+```
+
+## How to use
+
+### run nydusdserver
+```bash
+./nydusdserver $imagespath 
```

Comment:
Can we explain what the `$imagespath` means? How about changing to `/path/to/nydus/image/dir`.